website/docs/installation.mdx

```diff
@@ -302,43 +299,24 @@ kubectl create secret generic git-provider-credentials --namespace=wego-system \
 </TabItem>
 </Tabs>
 
-### 5. Upgrade
-
-<Tabs groupId="git-host" default>
-<TabItem value="hosted" label="github.com / gitlab.com">
-
-```bash
-gitops upgrade --version 0.0.18
-```
-
-</TabItem>
-<TabItem value="on-prem" label="GitLab on other domains">
-
-#### 5.1 Add `known_hosts` for other domains
+### 5. Install
 
-For hosts other than github.com or gitlab.com we need to retrieve the public ssh keys and save them in a secret. Replace `git.example.com` below with your git host.
+Commit and push all the files
 
-```bash
-ssh-keyscan git.example.com > known_hosts
-kubectl create configmap --namespace wego-system ssh-config --from-file=./known_hosts
-```
+### 5.5 Configure password and add rolebindings for the
 
-Install WGE and mount the `known_hosts` file:
+TODO: configure password
 
-```bash
-gitops upgrade \
-  --git-host-types="git.example.com=gitlab" \
-  --version 0.0.18 \
-  --set "config.extraVolumes[0].name=ssh-config" \
-  --set "config.extraVolumes[0].configMap.name=ssh-config" \
-  --set "config.extraVolumeMounts[0].name=ssh-config" \
-  --set "config.extraVolumeMounts[0].mountPath=/root/.ssh" \
-```
+By default a user `WegoAdmin` is used when you login.
```

Comment:
in core this is wego-admin, does WGE change this?

Reply:
No, bad typo, good catch

Reply:
```suggestion
By default a user `wego-admin` is used when you login.
```